ui/public/i18n/en/translation.json

```diff
@@ -55,7 +55,12 @@
         "days": "Days",
         "never": "Never clean",
         "hostname_pattern": "Must be a valid fully qualified domain name",
-        "hostname_format": "Must be a valid fully qualified domain name"
+        "hostname_format": "Must be a valid fully qualified domain name",
+        "lets_encrypt_tips": "This option enables automatic request of a Let's Encrypt TLS certificate for this app. Manage the certificate from the TLS Certificates page",
+        "cannot_obtain_certificate": "Cannot obtain a Let's Encrypt certificate for {hostname}",
+        "go_to_tls_certificates": "Go to TLS Certificates",
+        "lets_encrypt_disabled_warning_description": "The certificate might be removed. The app may temporarily become unreachable until a valid certificate is restored",
```

Comment:
I'd change this to: "The application will be temporarily unavailable, and HTTP clients connected to {node} will experience a brief disconnection.".

Also, remember to pass the node variable the translation function